src/server/components/formRunner/lawyers.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  Country,
+  LawyerListItemGetObject,
+  LawyerListItemJsonData,
+  ServiceType
+} from "server/models/types";
+import { FormRunnerFields, FormRunnerQuestion } from "server/components/formRunner/types";
+import { parseJsonFormData } from "server/components/formRunner/helpers";
+
+export async function generateFormRunnerWebhookData(listItem: LawyerListItemGetObject,
+                                                    listCountry?: Partial<Country>,
+                                                    isUnderTest?: boolean): Promise<Array<Partial<FormRunnerQuestion>>> {
+  const questions = await parseJsonFormData(ServiceType.lawyers, isUnderTest);
+
+  questions.forEach((question) => {
+    question.fields?.forEach((field) => {
+      const jsonData: LawyerListItemJsonData = listItem.jsonData ?? {};
+
+      switch (field.key) {
+        case FormRunnerFields.country :
+          field.answer = ((listCountry?.name) != null) ? listCountry.name : "";
```

Comment:
```suggestion
          field.answer = listCountry?.name ?? ""
```
`?.` <- if the left hand operand is truthy, continue, if it is null or undefined, stop executing and return null or undefined (whichever the lefthand operand is) 
`??` <- if the left hand operand is nullish (undefined or null), then return the righthand operand

so if either listCountry or listCountry.name is undefined, default to ""

Reply:
thanks for the pointer.  With your recommendation above, this point should then be redundant but useful for future.